### ISSUE TITLE
Allow microservices to configure event roles

### ISSUE THREAD

**Issue by plourenco** [indico/indico#4669]
When adding a custom service workflow to an event, we currently support basic feature such as updating tags and editing file types. Being able to update roles is an interesting use-case that can be combined with the custom actions in #4641.

The purpose of this issue is to allow the service to connect to the event endpoints to get/add/remove roles, similarly to tags.

**Comment by ThiefMaster** [comment]
Could indeed be interesting in the future. But I would wait whether people actually see a need for this and not spend time on implementing this on our own - at least not while event role management is not fully react+api based (in that case exposing the APIs would indeed be very easy).

**Comment by plourenco** [comment]
The openreferee Jacow version has the use case of checking if the user is in Scientific Secretary role before invoking a custom action.

Would be good if we could make sure the role is there by default by setting it ourselves.
Of course, we could always hard code the role code in the meantime or expect it to be set if you agree, but otherwise

**Comment by ThiefMaster** [comment]
Yeah, I think that's what we discussed: We hardcode a role code and expect the event organizers to set up this role. Failing to do so would simply not show QA to anyone, so it wouldn't cause any actual breakage.
